components/sidebar/sidebar.jsx

```diff
@@ -240,18 +237,26 @@ export default class Sidebar extends React.PureComponent {
     }
 
     setBadgesActiveAndFavicon() {
-        const link = document.querySelector('link[rel="icon"]');
-
-        if (!link) {
-            return;
-        }
-
         this.lastBadgesActive = this.badgesActive;
-        this.badgesActive = this.props.unreads.mentionCount > 0;
+        this.badgesActive = this.props.unreads.mentionCount;
```

Comment:
`this.badgesActive` is set to `false` later on in this file. Do you think it makes sense to do something like this?
```suggestion
        this.badgesActive = Boolean(this.props.unreads.mentionCount);
```